ereuse_devicehub/parser/snapshot.py

```diff
@@ -0,0 +1,49 @@
+from datetime import datetime, timezone
+from enum import Enum, unique
+from typing import List
+
+from ereuse_workbench.computer import Component, Computer, DataStorage
+from ereuse_workbench.utils import Dumpeable
+
+
+@unique
+class SnapshotSoftware(Enum):
+    """The algorithm_software used to perform the Snapshot."""
+
+    Workbench = 'Workbench'
+    AndroidApp = 'AndroidApp'
+    Web = 'Web'
+    DesktopApp = 'DesktopApp'
+
```

Comment:
Probably duplicated from `ereuse_devicehub/resources/enums.py`

Reply:
Is not in use. I drop it